src/wallet/wallet2.cpp

```diff
@@ -539,10 +540,11 @@ boost::optional<tools::password_container> get_password(const boost::program_opt
     return tools::password_container{command_line::get_arg(vm, opts.password)};
   }
 
-  if (command_line::has_arg(vm, opts.password_file))
+  if (!command_line::is_arg_defaulted(vm, opts.password_file))
   {
     std::string password;
-    bool r = epee::file_io_utils::load_file_to_string(command_line::get_arg(vm, opts.password_file),
+	const auto password_file = command_line::get_arg(vm, opts.password_file);
```

Comment:
Indentation

Reply:
Fixed, terribly sorry about all the indentation issues. I'll try to do a better job of noting what's being used for indentation in the future.